vmdb/app/models/miq_request.rb

```diff
@@ -502,4 +472,11 @@ def self.update_request(request, values, requester_id, target_class, event_messa
     return request
   end
 
+  private
+
+  def request_types
```

Comment:
@kbrock

1. doesn't look like it's called that often.
2. don't see any problem with it
3. love it :heart: